dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -191,9 +190,7 @@ private void validateUser(
       User user, Program program, ProgramStage programStage, OrganisationUnit requestedOrgUnit)
       throws ForbiddenException {
 
-    if (user == null
-        || user.isSuper()
-        || user.isAuthorized(F_TRACKED_ENTITY_INSTANCE_SEARCH_IN_ALL_ORGUNITS)) {
+    if (user == null || user.isSuper()) {
```

Comment:
This seems like a bug fixing.
Should it be here in this PR?

Reply:
I think so because this probably needs to be backported, right?

Reply:
it's possible yes